drools-wb-screens/drools-wb-guided-dtable-editor/drools-wb-guided-dtable-editor-client/src/main/java/org/drools/workbench/screens/guided/dtable/client/widget/analysis/action/FieldActionInspector.java

```diff
@@ -68,6 +70,31 @@ private boolean isValueRedundant(final DTCellValue52 other) {
         }
     }
 
+    private boolean valuesAreEqual(DTCellValue52 value, DTCellValue52 that) {
```

Comment:
Well, looking on `org.drools.workbench.models.guided.dtable.shared.model.DTCellValue52.equals()` implementation, I still think this `valuesAreEqual`method could be just `Objects.equals(value, that);`, however when I tried, the new added test fails, so probably keep as it is.

Reply:
The reason why it can not is that sometimes even when the value type is boolean (or anything, but String). If either object also has the stringValue field filled, it can be "" or null, the equals fails. Even though the real values are equal.

This is the original bug. Other having "" and other having null. So even if we detected this with boolean it does apply to everything, but the String.

Reply:
Yes, I noticed this during debug that boolean value and empty string was stored at the same time, I decided to not invest more time as it is on 6.5.x branch.